docs/resources/sso_connection.md

```diff
@@ -0,0 +1,71 @@
+---
+page_title: "castai_sso_connection Resource - terraform-provider-castai"
+subcategory: ""
+description: |-
+  SSO Connection resource allows creating SSO trust relationship with CAST AI.
+---
+
+# castai_sso_connection (Resource)
+
+SSO Connection resource allows creating SSO trust relationship with CAST AI.
+
+## Example Usage
+
+```terraform
+resource "castai_sso_connection" "sso" {
+  name         = "aad_connection"
+  email_domain = "aad_connection@test.com"
+  aad {
+    client_id     = azuread_application.castai_sso.client_id
+    client_secret = azuread_application_password.castai_sso.value
+    ad_domain     = azuread_application.castai_sso.publisher_domain
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `email_domain` (String) Email domain of the connection
+- `name` (String) Connection name
+
+### Optional
+
+- `aad` (Block List, Max: 1) Azure AD connector (see [below for nested schema](#nestedblock--aad))
+- `okta` (Block List, Max: 1) Okta connector (see [below for nested schema](#nestedblock--okta))
+- `timeouts` (Block, Optional) (see [below for nested schema](#nestedblock--timeouts))
```

Comment:
What is this timeouts, never seem this autogenerated

Reply:
We have those `timeouts` in most of our resource docs. It allows customizing the timeout values for a given resource - https://developer.hashicorp.com/terraform/plugin/sdkv2/resources/retries-and-customizable-timeouts